microceph/api/services.go

```diff
@@ -29,6 +31,33 @@ func cmdServicesGet(s *state.State, r *http.Request) response.Response {
 	return response.SyncResponse(true, services)
 }
 
+// Service Reload Endpoint.
+var restartServiceCmd = rest.Endpoint{
+	Path: "services/restart",
+	Post: rest.EndpointAction{Handler: cmdRestartServicePost, ProxyTarget: true},
+}
+
+func cmdRestartServicePost(s *state.State, r *http.Request) response.Response {
+	var services types.Services
+
+	err := json.NewDecoder(r.Body).Decode(&services)
+	if err != nil {
+		logger.Errorf("Failed decoding restart services: %v", err)
+		return response.InternalError(err)
+	}
+
```

Comment:
I'm thinking we should right at the boundary of our API service to user input handling so we know we get valid service names. Can we e.g. check that each of the services provided by the user are valid keys in our service table? Then we can later simplify our error handling because we know we sanitized user input at the boundary.